src/Serializable.ts

```diff
@@ -5,14 +5,21 @@ import { ParsingError } from "./parsing/ParsingError"
 import { SerializableBase } from "./SerializableBase"
 
 export class Serializable extends SerializableBase implements ISerializable {
+    /**
+     * Parses the given object to the class defined in `@type`.
+     *
+     * @param value the object to deserialize
```

Comment:
@ param value The JSON object to parse. The object must have the "@ type" property set in order to be parsed, in addition to possible "@ context" and "@ version" properties.

Reply:
Boaaah, hier muss man echt höllisch aufpassen. Es gibt Github user die param, type, context und version heißen und die verlinke ich die ganze Zeit :D